### ISSUE TITLE
Improvement: Apply correct formatting on code with multiple lines

### ISSUE THREAD

**Issue by jdev7** [aheze/Multiliner#6]
While it works perfectly on a one line function, if you already have code like this:
```swift
func foobar(_ foo: Foo,
                       bar: Bar)
```
and you apply multiliner, the result will be:
```swift
func foobar(
        _ foo: Foo,

                       bar: Bar
    )
```
which is not great. It would be awesome if it already detected new lines, and didn't apply new indentation. For the latter I don't know if it could re-use Xcode's reindent command for each line maybe, so you don't have to implement it yourself.

**Comment by aheze** [comment]
Hmm, I think I can fix this by stripping newlines before piecing it back together. I'll get back to this tomorrow, thanks for the suggestion!

**Comment by aheze** [comment]
@jdev7 fixed in 1.1! https://github.com/aheze/Multiliner/releases/tag/1.1.0

**Comment by jdev7** [comment]
Works perfect! Thanks :)
